test/functional/fixtures/regression/gh-993/testcafe-fixtures/index.test.js

```diff
@@ -0,0 +1,20 @@
+import { ClientFunction } from 'testcafe';
+import { expect } from 'chai';
+
+fixture `t.pressKey on angular2 element`
+    .page('http://localhost:3000/fixtures/regression/gh-993/pages/index.html');
+
+test("Press the 'enter' key on the input", async t => {
+    await t
+        .click('#target')
+        .pressKey('enter');
+
+    const getKeySequences = ClientFunction(()=> {
+        return { keydown: window.keydownSequence, keyup: window.keyupSequence };
+    });
+
+    const { keydown, keyup } = await getKeySequences();
+
+    expect(keydown).equal('Enter');
+    expect(keyup).equal('Enter');
```

Comment:
Original page doesn't check keypress event. As well Safari doesn't fill 'keyIdenfier' property in keypress event while pressing 'enter'. There is ambiguity, i acknowledge, but i decided to leave it as is, cause it will completed [asap] (https://github.com/georgiy-abbasov/testcafe-phoenix/commit/4e18b45ac589a263735045673a3eb07c2cc2e441)